### PR TITLE
[v6r21] RSS FIX: fix reporting from EmailAgent

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*RSS
FIX: fix reporting from EmailAgent

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
You did it against integration, is that on purpose ?

**Comment by fstagni** [comment]
Ah no... corrected, sorry.

File: ResourceStatusSystem/PolicySystem/Actions/EmailAction.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def run(self):
       elif not siteName['Value']:
         siteName = "Unassigned Resources"
       else:
-        siteName = siteName['Value'][0]
+        siteName = siteName['Value'] if isinstance(siteName['Value'], basestring) else siteName['Value'][0]
```
Comment (chaen) [inline ResourceStatusSystem/PolicySystem/Actions/EmailAction.py]: isnt' it simpler to take the first of the list in `getSitesForSE` a bit higher ? Then you don't need any condition here

Comment (fstagni) [reply ResourceStatusSystem/PolicySystem/Actions/EmailAction.py]: Nah, it's more modifications.

**Pull request merged**